zipline/assets/_assets.pyx

```diff
@@ -42,25 +39,35 @@ cdef class Asset:
 
     cdef readonly object exchange
 
+    cdef readonly object ccy
+    cdef readonly object price_format
+    cdef readonly object status
+
     def __cinit__(self,
-                  int sid, # sid is required
+                  int sid,  # sid is required
                   object symbol="",
                   object asset_name="",
                   object start_date=None,
                   object end_date=None,
                   object first_traded=None,
                   object exchange="",
+                  object ccy="",
+                  object price_format=0,
+                  object status="",
                   *args,
                   **kwargs):
 
-        self.sid           = sid
-        self.sid_hash      = hash(sid)
-        self.symbol        = symbol
-        self.asset_name    = asset_name
-        self.exchange      = exchange
-        self.start_date    = start_date
-        self.end_date      = end_date
-        self.first_traded  = first_traded
+        self.sid = sid
+        self.sid_hash = hash(sid)
+        self.symbol = symbol
+        self.asset_name = asset_name
+        self.exchange = exchange
+        self.start_date = start_date
+        self.end_date = end_date
+        self.first_traded = first_traded
+        self.ccy = ccy
+        self.price_format = price_format
+        self.status = status
```

Comment:
status is time varying perhaps in a number of ways. a stock could be delisted and re-listed or the symbol could be re-used but our sid would always stay constant. this may have to be handled like the splits are in a seperate table and perhaps there would be multiple of these both on the exchange,index or other indice levels?